DistributionModels/weibull.py

```diff
@@ -84,27 +107,67 @@ def wscore(self, distances):
         smallScoreTensor=self.smallScoreTensor
         if len(self.smallScoreTensor.shape)==2:
             smallScoreTensor=self.smallScoreTensor[:,0]
-        distances = distances + 1 - smallScoreTensor.to(self.deviceName)[None,:]
+            ## tb hack 
```

Comment:
The hack is to change the `self.translateAmountTensor` tensor which is done at the constructor level and not here, so probably should remove this comment